state/secrets_test.go

```diff
@@ -927,39 +955,79 @@ func (s *SecretsSuite) TestUpdateConcurrent(c *gc.C) {
 }
 
 func (s *SecretsSuite) TestChangeSecretBackend(c *gc.C) {
+	backendStore := state.NewSecretBackends(s.State)
+	_, err := backendStore.CreateSecretBackend(state.CreateSecretBackendParams{
```

Comment:
Can we also retain a test where the original backend is the internal one, ie add a new test for the case where the original backend is external.